frontend/src/app/main/ui/comments.scss

```diff
@@ -98,8 +98,12 @@
 }
 
 .content {
+  position: relative;
   @include bodySmallTypography;
   color: var(--color-foreground-primary);
+  word-wrap: break-word;
+  overflow-wrap: break-word;
```

Comment:
We need to do this in addition to `hyphens` auto in the cases the browser-dependent algorithm does not break the word: comments in different languages than the one specified in our `<html lang>` attribute, made up words (for instance, a very long string with just `a`'s, etc.

Reply:
Didn't know the `hyphens` property. Very interesting.